### PR TITLE
Deleted duplicated meshes from URDF directories

### PR DESCRIPTION
**Pull request by dtdavi1**
Updated description packages for the DVLs and plug and socket to use the
meshes from the SDF models instead of duplicating them (DVL models are
in the dave_sensor_models package, and the plug and socket models are in
the dave_object_models package).

### CODE REVIEW

**Comment by dtdavi1** [comment]
Pull request is associated with issue [#151](https://github.com/Field-Robotics-Lab/dave/issues/151).

**Review by mabelzhang — COMMENTED** [review]
Two CMakeLists.txt files have typos. Otherwise I tested to see that the new model paths show up.

Since the `install()` commands are updated, I tested with `catkin_make install`. It errors for non-existent `meshes` directories, which still need to be removed from the `install()` command for `teledyne_whn_dsl_description` and `plug_and_socket_description` packages.

I tested the new `.dae` paths in two ways:
- `ls` to make sure all the new paths exist in `models`
- `grep` to trace each new `dae` path up to the top-level launch files that will load the model in Gazebo GUI. Ran the launch files and made sure models showed up.

Specifically, these are the launch files I tested to see the models show up in Gazebo GUI:
```
roslaunch dave_demo_launch dave_plug_and_socket_demo.launch
roslaunch dave_demo_launch dave_dvl_gradient_demo_dsl.launch
roslaunch dave_demo_launch dave_uuvsim_gradient_demo_dsl.launch
roslaunch dave_sensor_launch dvl_side_by_side_dsl.launch
roslaunch dave_sensor_launch dvl_side_by_side_uuvsim.launch
```

Some rosparam-setting launch files aren’t used in any top-level launch files, so I didn’t check visually in the GUI:
```
upload_teledyne_explorer1000.launch
upload_nortek_dvl1000_300.launch
upload_sonardyne_syrinx600.launch
upload_nortek_dvl500_6000.launch
upload_nortek_dvl500_300.launch
```

Let me know if I missed anything. This was a good exercise for me. I had never run these sensors before!

File: urdf/sensors/teledyne_whn_dsl_description/CMakeLists.txt [hunk 1]
```diff
@@ -12,8 +12,3 @@ install(DIRECTORY meshes urdf launch
         DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
         PATTERN "*~" EXCLUDE)
 
-
-install(DIRECTORY launch meshes urdf
```
[the remark this answers is not in the document]
Comment (mabelzhang) [reply urdf/sensors/teledyne_whn_dsl_description/CMakeLists.txt:16]: Same for `plug_and_socket_description` package.

**Comment by dtdavi1** [comment]
@mabelzhang The CMakeLists.txt files files have been updated if you want to give it another quick look and approve the merge (if nothing else is broken).

**Review by mabelzhang — APPROVED** [review]

**Pull request merged**